Sample-01/src/app/components/nav-bar/nav-bar.component.ts

```diff
@@ -1,20 +1,26 @@
 import { Component, OnInit } from '@angular/core';
 import { faUser, faPowerOff } from '@fortawesome/free-solid-svg-icons';
-import { AuthService } from 'src/app/auth/auth.service';
+import { AuthService } from '@auth0/auth0-angular';
 
 @Component({
   selector: 'app-nav-bar',
   templateUrl: './nav-bar.component.html',
-  styleUrls: ['./nav-bar.component.css']
+  styleUrls: ['./nav-bar.component.css'],
 })
 export class NavBarComponent implements OnInit {
   isCollapsed = true;
   faUser = faUser;
   faPowerOff = faPowerOff;
 
-  constructor(public auth: AuthService) { }
+  constructor(public auth: AuthService) {}
 
-  ngOnInit() {
+  ngOnInit() {}
+
+  loginWithRedirect() {
+    this.auth.loginWithRedirect();
   }
 
+  logout() {
+    this.auth.logout({ returnTo: window.location.origin });
```

Comment:
the article uses `http://localhost:4200`. And here everywhere we use `window.location.origin`. I think on either side we should anticipate this. 

e.g. Adding in the article "here we use localhost, but on the sample you'll notice we use window.location.origin. Which will default to X value".

Reply:
Sounds good - let me add that note into the article 👍